.github/workflows/main_push_and_pull_request_workflow.yml

```diff
@@ -24,13 +24,62 @@ jobs:
           java-version: ${{ matrix.java-version }}
           distribution: temurin
 
+      - name: Read version from properties file
+        id: read-version
+        run: |
+          version=$(grep -oP 'version=\K[^[:space:]]+' gradle.properties)
+          echo "Version found: $version"
+          echo "VERSION=$version" >> $GITHUB_ENV
```

Comment:
I wonder if we can integrate this into the Makefile, e.g. create a phony target that builds the tgz of the current version. It would make it unnecessary to have these steps on the CI.

Reply:
Agree. Was thinking about this, will give a try.

Reply:
Done